### ISSUE TITLE
Document the release cycle

### ISSUE THREAD

**Issue by lab-at-nohl** [tine20/tine20#7351]
I noticed the nightly builds. However, I do not see the purpose of it. Will regular versioned releases (20yy.mm.x) remain? How to assess stability/progress in nightly builds?

Could you maintain a small wiki page with major builds and a short description of the release cycle? Monitor releases is not enough anymore.

**Comment by pschuele** [comment]
hi, 
thanks for the questions. we will add a wiki page and a discussion about this topic soon.

**Comment by pschuele** [comment]
hi,
here's an update on the changes to our release cycle (not everything is set in stone at the moment and might change):

- we release a weekly version each Saturday (version: YYYY.MM.DD.numberOfCommitsSinceLastRelease)
- we release an "update version" for the defined update path once a year (it is always possible to update from the last "update path" to the next or the current version) version string/tag: YYYY.MM.0.0
- we no longer build distribution packages (see https://packages.tine20.org)

we'll update the wiki as soon as this is finalized.

**Comment by msphn** [comment]
Also maybe worth to mention that there are finally reasonably priced SAAS packages. This also fits for private usage https://www.tine-groupware.de/produkt/groupware/

You could have told me earlier, I just migrated my stuff to another provider because I got tired of managing all by myself  😭

**Comment by msphn** [comment]
I don't use twitter :/

**Comment by msphn** [comment]
https://github.com/tine20/tine20/releases/tag/2022.08.13.154

Seems to work

**Comment by ravermeister** [comment]
yep, but between the [last version](https://github.com/tine20/tine20/releases/tag/2022.08.13.154) `2022.08.13.154`  and the [former version](https://github.com/tine20/tine20/releases/tag/2022.07.23.83) `2022.07.23.83` 22 days (more than a week...) passed ;)

anyway not a big problem, and thanks for the Update! :)
